### ISSUE TITLE
[Map Local] #8 Auto Completion

### ISSUE THREAD

**Issue by NghiaTranUIT** [ProxymanApp/proxyman-windows-linux#217]
## Description
- ⚠️ This Ticket should be implemented after #193 is merged into the main. 
- Write auto-completion for the first line (HTTP Protocol, Status Code, Header and Value) for Monaco Editor
- ✅ Ping @NghiaTranUIT to provide the Monaco Autocomplete provider, which he's already ship on the macOS.

## Acceptance Criteria
- Use Monaco Editor
- Able to show auto-completion for HTTP Protocol, Status Code, Header, and Value (Check the macOS Map Local)

### If you reuse the AutoCompletion Provider from the macOS, we will get these features for free.
- HTTP Protocol, Status Code should only auto-completed if the cursor is in the first line
- Header auto-completion only shows if it's in the Header Section
- Value auto-completion only shows if it's in the Value Section
